extension-requests/script.js

```diff
@@ -817,6 +862,48 @@ async function createExtensionCard(data) {
     extensionForValue.classList.toggle('hidden');
     extensionInput.classList.toggle('hidden');
   }
+
+  async function renderLogs({ extensionRequestId, assigneeName, createdAt }) {
+    const logContainer = document.getElementById(
+      `log-container-${extensionRequestId}`,
+    );
+    if (logContainer.querySelector('.log-div')) {
+      return;
+    }
+
+    let creationLog = document.createElement('div');
+    creationLog.classList.add('log-div');
+
+    let logImg = document.createElement('img');
+    logImg.classList.add('log-img');
+    logImg.src = '/images/calendar-plus.png';
+
+    let logText = document.createElement('p');
+    logText.classList.add('reason-text');
+    logText.innerText = `${assigneeName} has created this extension request on ${fullDateString(
+      secondsToMilliSeconds(createdAt),
+    )}.`;
+
+    creationLog.appendChild(logImg);
+    creationLog.appendChild(logText);
+
+    logContainer.appendChild(creationLog);
+
+    const extensionLogs = await getExtensionRequestLogs({
+      extensionRequestId,
+      isDev: true,
+    });
+    if (
+      extensionLogs.logs[0]?.body?.newEndsOn &&
```

Comment:
what if extensionLogs is null

Reply:
Removed this redundant code as it is of no use